xline/src/storage/kv_store.rs

```diff
@@ -140,31 +142,33 @@ where
     }
 
     /// Get KV watcher
-    pub(crate) fn kv_watcher(&self) -> Arc<KvWatcher<S>> {
+    pub(crate) fn kv_watcher(&self) -> Arc<KvWatcher<DB>> {
         Arc::clone(&self.kv_watcher)
     }
 }
 
-impl<S> KvStoreBackend<S>
+impl<DB> KvStoreBackend<DB>
 where
-    S: StorageApi,
+    DB: StorageApi,
 {
     /// New `KvStoreBackend`
     pub(crate) fn new(
         kv_update_tx: mpsc::Sender<(i64, Vec<Event>)>,
         lease_cmd_tx: mpsc::Sender<LeaseMessage>,
         header_gen: Arc<HeaderGenerator>,
-        storage: S,
-    ) -> Self {
-        Self {
+        db: Arc<DB>,
+    ) -> Result<Self, ExecuteError> {
+        db.create_table("kv")?;
```

Comment:
Our table will hardly change, I think we can use a set of global `&'static str` or an enumeration to represent all tables, and collect them into an array, and create all tables when creating Engine, like this There will be no errors here. When we use RocksDB, opening an existing DB also needs to provide slices containing all Cf names

Reply:
Fixed